src/components/BaseSearch.vue

```diff
@@ -57,7 +68,8 @@ function clearInput() {
       class="input w-full flex-auto border-none"
       @input="handleInput"
     />
-    <a @click="clearInput">
+    <slot name="after" :clearInput="clearInput" />
+    <a v-if="!$slots.after" @click="clearInput">
```

Comment:
We should use <i-ho-name /> like explained here https://www.notion.so/snapshotlabs/Getting-started-with-Snapshot-Ui-3ff7be7be0ac435797f198199d4f6e6e?pvs=4

BaseIcon is the old way that we still haven't fully refactored out of our code.